### PR TITLE
Update Gradle dependencies

### PR DESCRIPTION
**Pull request by davidbailey00**
Updates the following dependencies:

- Yarn: `1.16.5+build.5` → `1.16.5+build.6`
- Fabric: `0.31.0+1.16` → `0.32.5+1.16`
- Fabric Loader: `0.11.2` → `0.11.3`

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,5 +1,5 @@
 plugins {
-	id 'fabric-loom' version '0.6-SNAPSHOT'
+	id 'fabric-loom' version '0.7-SNAPSHOT'
```
Comment (ByMartrixx) [inline build.gradle:2]:
```suggestion
	id 'fabric-loom' version '0.6-SNAPSHOT'
```
As far as I know, Loom 0.7 is in development and still is unstable, it should not be used

File: gradle.properties [hunk 2]
```diff
@@ -4,7 +4,7 @@ org.gradle.jvmargs=-Xmx1G
 # Fabric Properties
 	# check these on https://fabricmc.net/use
 	minecraft_version=1.16.5
-	yarn_mappings=1.16.5+build.5
+	yarn_mappings=1.16.5+build.6
 	loader_version=0.11.2
```
Comment (ByMartrixx) [inline gradle.properties:8]:
```suggestion
	loader_version=0.11.3
```

File: build.gradle (hunk 1 above)
Comment (modmuss50) [inline build.gradle:2]: No this isn’t stable yet

**Review by modmuss50 — CHANGES_REQUESTED** [review]

**Review by ByMartrixx — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (davidbailey00) [reply build.gradle]: Thanks, I've reverted this

**Review by ByMartrixx — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**